lib/jsonapi/authorization/default_pundit_authorizer.rb

```diff
@@ -146,8 +146,16 @@ def replace_to_one_relationship(_source_record, _old_related_record, _new_relate
       #
       # * +source_record+ - The record whose relationship is modified
       # * +new_related_records+ - The new records to be added to the association
-      def create_to_many_relationship(_source_record, _new_related_records)
-        raise NotImplementedError
+      # * +relationship_type+ - The relationship type
+      def create_to_many_relationship(source_record, new_related_records, relationship_type)
+        policy = Pundit.policy(user, source_record)
+        relationship_method = 'allow_relationship_' + relationship_type.to_s + '?'
+        if policy.respond_to?(relationship_method)
+          allowed = policy.send(relationship_method, new_related_records)
+        else
+          allowed = policy.update?
+        end
```

Comment:
The build is currently failing because [Rubocop](https://github.com/bbatsov/rubocop) would like this piece of code to look something like this:

```rb
allowed =
  if policy.respond_to?(relationship_method)
    policy.send(relationship_method, new_related_records)
  else
    policy.update?
  end
```